src/components/number-input/number-input.css

```diff
@@ -0,0 +1,71 @@
+@value vars: "../../style/default-theme/variables.css";
+@value borderDefault, borderWidth, borderHover, color_MainTextBackground, color_MainText from vars;
+
+.number-input {
+    position: relative;
+    display: inline-block;
+}
+
+.stepper {
+    position: relative;
+
+    width: 36px;
+    height: 36px;
```

Comment:
Not originally. I was thinking to add `inputHeight` variable and use it as `width: inputHeight;` but hesitated.